homeassistant/components/light/lutron.py

```diff
@@ -0,0 +1,99 @@
+"""Support for Lutron lights."""
+import asyncio
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.components.lutron import (
+    LutronDevice, LUTRON_DEVICES, LUTRON_GROUPS, LUTRON_CONTROLLER)
+
+DEPENDENCIES = ['lutron']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
```

Comment:
Re __init__, that seems like a hack though, having to set hass inside the entity manually? But I'm not above such hacks if they are sanctioned by the maintainers :)

Re add_devices(), I'd like to query all the devices to get their state at startup. Without adding True to add_devices and calling update() manually in my device init, how will the new state be queried?

Thanks for all the feedback!